org.adempiere.base/src/org/compiere/model/MDocumentStatus.java

```diff
@@ -58,17 +58,21 @@ public MDocumentStatus(Properties ctx, ResultSet rs, String trxName) {
 	 * @param AD_Role_ID 
 	 *	@return array of document status
 	 */
-	public static MDocumentStatus[] getDocumentStatusIndicators(Properties ctx, int AD_User_ID, int AD_Role_ID)
+	public static MDocumentStatus[] getDocumentStatusIndicators(Properties ctx, int AD_User_ID, int AD_Role_ID, String trxName)
```

Comment:
our standard practice is to maintain the existing method signature and call the new method signature will null trxname. also, javadoc is not updated for the new method signature.